pygeo/parameterization/DVGeoCST.py

```diff
@@ -286,7 +286,7 @@ def addPointSet(self, points, ptName, boundTol=1e-10, **kwargs):
         boundTol : float, optional
             Small absolute deviation by which the airfoil coordinates can exceed the initial
             minimum and maximum x coordinates, by default 1e-10.
-        kwargs
+        \*\*kwargs
```

Comment:
I would be surprised if backslash is necessary here

Reply:
Yes, that was my thought too. However, if removed, the following is generated 
`RST210 Inline strong start-string without end-string`.  There is an open issue on this https://github.com/peterjc/flake8-rst-docstrings/issues/18 and some PR, but not sure why its not merged yet. While escaping is ugly, it works and is consistent with rst.

Reply:
I see. I guess I'm fine with this then, even though it is harder to read. The alternative would be to ignore RST210 but that may mask other issues.